static/scss/partials/main.scss

```diff
@@ -676,7 +676,7 @@ table {
   height: 100%; /* expand clickable area of relay address */
   display: block;
   position: relative;
-  max-width: 100%;
+  max-width:100%;
```

Comment:
nit: white space (which would mean no change to this line)

Reply:
Updated the white space issues!